pandas/tests/exchange/test_impl.py

```diff
@@ -0,0 +1,165 @@
+import random
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+from pandas.core.exchange.dataframe_protocol import (
+    ColumnNullType,
+    DtypeKind,
+)
+from pandas.core.exchange.from_dataframe import from_dataframe
+
+test_data_categorical = {
+    "ordered": pd.Categorical(list("testdata") * 30, ordered=True),
+    "unordered": pd.Categorical(list("testdata") * 30, ordered=False),
+}
+
+NCOLS, NROWS = 100, 200
+
+int_data = {
+    f"col{int((i - NCOLS / 2) % NCOLS + 1)}": [
+        random.randint(0, 100) for _ in range(NROWS)
```

Comment:
Am I missing something or there's no fixed initial state specified for `random`? Better add one just for safety :smile: